doc_src/getting_started.md

```diff
@@ -34,6 +34,6 @@
 - Congratulations! Now your first gear is up and running.
     - Take a look around and play around with it.
     - Check out documentations and see what you can do with antikythera and gears.
-        - [Guide for instance administrators](./instance_administrators/) (TBD)
-        - [Guide for gear developers (i.e. developers of web services)](./gear_developers/)
+         - Guide for instance administrators[](./instance_administrators/) (TBD)
```

Comment:
Sorry, my suggestion seems to have included extra white space.
```suggestion
        - Guide for instance administrators[](./instance_administrators/) (TBD)
```

Reply:
I applied
120ddef37e199f7d181c71e5f59909767771204a